### PR TITLE
Background button

### PR DESCRIPTION
**Pull request by TimNguyen21**
Created Background Button Feature to show different background when selecting and clicking a button.

- Added functionality to buttons within the background section
- Added classes to buttons on the background section

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -223,6 +223,7 @@ display: inline-block;
   width: 95%;
   height: 85%;
   justify-content: space-around;
+  border-radius: 4em;
```
Comment (TimNguyen21) [inline styles.css:226]: This addition added curved corners of the Main Bear image.

File: index.html [hunk 2]
```diff
@@ -42,12 +42,12 @@ <h3 clothes-title><span class="arrows">➫</span>Accessories</h3>
...
-            <button class="favoriteStyled background-Options space margin-left-class-btn" type="button">Space</button>
-            <button class="favoriteStyled background-Options yellow" type="button">Yellow</button>
-            <button class="favoriteStyled background-Options hearts margin-left-class-btn" type="button">Hearts</button>
+            <button class="favoriteStyled background-Options blue blue-background" type="button">Blue</button>
+            <button class="favoriteStyled background-Options park park-background margin-left-class-btn" type="button">Park</button>
+            <button class="favoriteStyled background-Options beach beach-background margin-left-class-btn" type="button">Beach</button>
+            <button class="favoriteStyled background-Options space space-background margin-left-class-btn" type="button">Space</button>
+            <button class="favoriteStyled background-Options yellow yellow-background" type="button">Yellow</button>
+            <button class="favoriteStyled background-Options hearts heart-background margin-left-class-btn" type="button">Hearts</button>
```
Comment (TimNguyen21) [inline index.html:50]: Added more specific class name.  Can be changed in future if necessary.

Comment (Corbinj22) [reply index.html:50]: Like it. Let's discuss today what we want to do with the converged classes we meshed.

**Review by Corbinj22 — COMMENTED** [review]
@infamouskeyduster looks good to me. If you want to once over and approve. My only suggestion is that today we get together and find a common ground on the classes, then reform the CSS changes to reflect the correct class.

File: index.html (hunk 2 above)
Comment (infamouskeyduster) [reply index.html:50]: Yep, agreed. Lets make sure that we have clear & definitive classes to each element / One class that the CSS interacts with, and One class that the JavaScript interacts with (as needed).

**Pull request merged**